tools/gfastats/gfastats.xml

```diff
@@ -0,0 +1,298 @@
+        #else
+            #if $mode_condition.statistics_condition.selector == 'size'
+                --out-size $mode_condition.statistics_condition.out_size
+            #else if $mode_condition.statistics_condition.selector == 'coordinates'
+                --out-coord $mode_condition.statistics_condition.out_coord
+            #else if $mode_condition.statistics_condition.selector == 'assembly'
+                --nstar-report
+            #else
+                --seq-report
+                $mode_condition.statistics_condition.out_sequence
+            #end if
+        #end if
+        $tabular > $stats 
+        #if $mode_condition.selector == 'manipulation'
+            && mv dataset* output_dataset
+        #end if
+    ]]></command>
+    <inputs>
+        <param name="input_file" argument="--fasta" type="data"
+            format="fasta,fastq,fastqsanger,gfa1,fasta.gz,fastq.gz,fastqsanger.gz,gfa1.gz"
+            label="Input file"/>
+        <conditional name="target_condition">
+            <param name="target_option" type="select" label="Specify target sequences">
+                <option value="false">Disabled</option>
+                <option value="true">Enabled</option>
+            </param>
+            <when value="false"/>
+            <when value="true">
+                <param name="target_sequence" type="text" value="" label="Target sequence" help="Target specific sequence by header, optionally with coordinates: header[:start-end]">
+                    <sanitizer invalid_char="">
+                        <valid initial="string.digits,string.letters">
+                            <add value=":"/>
+                            <add value="-"/>
+                            <add value="_"/>
+                            <add value="|"/>
+                            <add value=" "/>
+                        </valid>
+                    </sanitizer>
+                    <validator type="regex">[0-9A-Za-z:-_| ]+</validator>
+                </param>
+                <param argument="--include-bed" type="data" optional="true"
+                    format="bed" label="Include specific intervals"
+                    help="Generates output on a subset list of headers or coordinates   
+                        in 0-based bed format. It can be combined with --exclude-bed. Optional"/>
+                <param argument="--exclude-bed" type="data" format="bed" optional="true"
+                    label="Exclude specific intervals"
+                    help="Exclude a subset of headers or coordinates in 0-base bed format. It can be conmbined with --include-bed Optional"/>
+            </when>
+            <when value="false"/>
+        </conditional>
+        <conditional name="mode_condition">
+            <param name="selector" type="select" label="Tool mode">
+                <option value="manipulation">Genome assembly manipulation</option>
+                <option value="statistics">Summary statistics generation</option>
```

Comment:
is stats maybe the preferred option, the most used option? then add it as first option

Reply:
Yes, probably it is the preferred one.